src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -71,6 +74,27 @@ private static boolean isGoogModuleCall(Node statement) {
     return expression.isCall() && expression.getFirstChild().matchesQualifiedName("goog.module");
   }
 
+  /** Matches a top level `goog.require()` call. */
+  private static boolean isGoogRequireCall(Node statement) {
+    if (statement == null) {
+      return false;
+    }
+
+    if (!statement.isExprResult()) {
+      return false;
+    }
+
+    if (statement.getFirstChild() == null) {
+      return false;
+    }
+
+    Node googRequireCall = statement.getFirstChild();
+
+    return googRequireCall != null
```

Comment:
You null-check the first child twice.  I think maybe you should drop the getFirstChild() == null "if" block.

Reply:
Whoops - done!